### PR TITLE
Feature: create project model for deployment process

### PR DESCRIPTION
**Pull request by ShellyXueHan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "express": "^4.16.3",
     "express-session": "^1.15.6",
     "fs-extra": "^7.0.0",
-    "handlebars": "^4.0.11",
+    "handlebars": "^4.0.12",
```
Comment (jleach) [inline package.json:23]: I don't think we use this package for anything. I think we can remove it.

File: src/router/routes/deploy.js [hunk 2]
```diff
@@ -40,22 +40,22 @@ import shared from '../../libs/shared';
...
 const router = new Router();
 const dm = new DataManager();
-const { db, Job } = dm;
+const { db, Job, Project } = dm;
 const bucket = config.get('minio:bucket');
 
 router.post(
   '/:jobId',
   asyncMiddleware(async (req, res) => {
     const { jobId } = req.params;
```
Comment (jleach) [inline src/router/routes/deploy.js]: Can we ax this space?

**Review by jleach — APPROVED** [review]
Some changes to consider.

File: package.json (hunk 1 above)
Comment (ShellyXueHan) [reply package.json:23]: it's a dependency in jest cli.

**Pull request merged**